apps/rosy-design-system/src/app/popover.component.ts

```diff
@@ -13,8 +13,14 @@ import { Popover } from '@rosy-design-system';
         <option value="bottom">Bottom</option>
         <option value="left">Left</option>
         <option value="right">Right</option>
-        <option value="center-left">Center Left</option>
-        <option value="center-right">Center Right</option>
+        <option value="top-start">Top Start</option>
+        <option value="top-end">Top End</option>
+        <option value="bottom-start">Bottom Start</option>
```

Comment:
Positions working properly. But facing an issue in @container style() , where all styles are getting applied. Please see the style section in below image
![image](https://github.com/user-attachments/assets/ff8546e2-4f4d-4b93-8fc9-5692b4d9ad94)

It seems even when we change the css initial value, it is not getting reflected in build. Please check
![image](https://github.com/user-attachments/assets/aca1292b-391c-47e2-8f6d-0d0f6ac3470a)

Reply:
Will check

Reply:
Thanks for contributing.